mkdocs.yml_template

```diff
@@ -99,7 +99,7 @@ nav:
     - "GUI Extension API":
       - manuals/reference_guiext/index.md
       - "taipy-gui module": manuals/reference_guiext/modules.md
-  - "Upgrades": 'manuals/legacy-version-migration.md'
+  - "Upgrades": 'legacy-version-migration.md'
```

Comment:
Actually I would not give access to this page from the very top.
New users will not start by upgrading or migrating.
Legacy users will know that the Release Notes will help.

And room in the menu bar is precious...

Reply:
"Lopin com pru.". I don't understand what you would like.

Reply:
We're good now.